pkg/utils/utils.go

```diff
@@ -258,3 +258,46 @@ func CleanCachedNetConf(cRefPath string) error {
 	}
 	return nil
 }
+
+func SaveAllocatedPCI(pciAddress, ns, dataDir string) error {
+	allocatedPCIPath := filepath.Join(dataDir, "pci")
+	if err := os.MkdirAll(allocatedPCIPath, 0700); err != nil {
+		return fmt.Errorf("failed to create the sriov data directory(%q): %v", dataDir, err)
+	}
+
+	path := filepath.Join(allocatedPCIPath, pciAddress)
+
+	err := os.WriteFile(path, []byte(ns), 0700)
```

Comment:
I suggest to grant this file only read and write permissions.

```suggestion
	err := os.WriteFile(path, []byte(ns), 0600)
```